lib/versioning/pvp/index.ts

```diff
@@ -0,0 +1,80 @@
+import { regEx } from '../../util/regex';
+import { GenericVersion, GenericVersioningApi } from '../loose/generic';
+import { VersioningApi } from '../types';
+
+export const id = 'pvp';
+export const displayName = 'PVP';
+export const urls = ['https://pvp.haskell.org'];
+export const supportsRanges = false;
+
+/**
+ * At least 3 components and no leading 0s
+ * https://pvp.haskell.org/#version-number
+ */
+const versionPattern = regEx(
+  /^((([0-9])|([1-9][0-9]*))\.){2,}(([0-9])|([1-9][0-9]*))$/
+);
+
+class PvpVersioningApi extends GenericVersioningApi {
+  /**
+   * PVP has two major components A.B
```

Comment:
Is it clear to you programmers what `A.B` refers to? To me, it feels like this comment doesn't really explain it properly.

Reply:
That makes sense. I was referring to the spec which mentions those components https://pvp.haskell.org/#version-number

What do you think about:
> The first two components in PVP are the major numbers
e.g. 1.2.3.4 has a major version of 1.2

Reply:
Something like that will probably be easier to understand, but I'm not sure your proposed sentence explains both `A` _and_ `B`?

Reply:
Would quoting the specification be better: https://pvp.haskell.org/

> A package version number SHOULD have the form A.B.C, and MAY optionally have any number of additional components, for example 2.1.0.4 (in this case, A=2, B=1, C=0). 
> A.B is known as the major version number, and C the minor version number.